clang/unittests/Format/FormatTest.cpp

```diff
@@ -18525,12 +18525,17 @@ TEST_F(FormatTest, AlignConsecutiveAssignments) {
                "                     a_longer_name_for_wrap}};",
                Alignment);
 
-  Alignment.ColumnLimit = 60;
+  Alignment = getLLVMStyleWithColumns(60);
+  Alignment.AlignConsecutiveAssignments.Enabled = true;
```

Comment:
Is this necessary? If `AlignConsecutiveAssignments` is already on, you can use `EXPECT_TRUE` instead.

Reply:
Nvm.